mono/tests/test-runner.cs

```diff
@@ -219,6 +238,14 @@ class TestInfo {
 							Console.Write (".");
 						passed.Add(process_data [dead]);
 						npassed ++;
+					// FIXME: is this valid for all targets?
+					} else if (dead.ExitCode == 137) {
```

Comment:
I suppose it won't work on Windows, but I don't think we run the runtime tests there anyway :smile: